x-template.js

```diff
@@ -94,8 +94,8 @@ class TemplateEngine {
    * html`<a href="${ifDefined(obj.href)}"></div>`;
    * ```
    * @deprecated
-   * @param {any} value
-   * @returns {any}
+   * @param {unknown} value
```

Comment:
Here, the semantics are now _much better_. It’s not that this is _any_, rather, we just say “we dunno what this is, so we’ll force ourselves to check.”